src/wagtail_personalisation/locale/en/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,320 @@
+# Wagtail Personalisation english translation strings.
+# Copyright (C) 2017 Lab Digital B.V.
+# This file is distributed under the same license as the wagtail_personalisation package.
+# Boris Besemer <b.besemer@labdigital.nl>, 2017.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: wagtail_personalisation 0.1.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-05-31 03:00-0500\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: blocks.py:19
+msgid "Personalisation segment"
+msgstr ""
+
+#: blocks.py:20
+msgid "Only show this content block for users in this segment"
+msgstr ""
+
+#: config.py:8
+msgid "Wagtail Personalisation"
+msgstr ""
+
+#: models.py:29
+msgid "Enabled"
+msgstr ""
+
+#: models.py:30
+msgid "Disabled"
+msgstr ""
+
+#: models.py:35
+msgid "Should the segment persist between visits?"
+msgstr ""
+
+#: models.py:38
+msgid "Should the segment match all the rules or just one of them?"
+msgstr ""
+
+#: models.py:56
+msgid "Rules"
+msgstr ""
+
+#: models.py:137
+msgid "Content"
+msgstr ""
+
+#: models.py:139
+msgid "Variations"
+msgstr ""
+
+#: models.py:141
+msgid "Promote"
+msgstr ""
+
+#: models.py:143
+msgid "Settings"
+msgstr ""
+
+#: rules.py:30 rules.py:45
+msgid "Abstract segmentation rule"
+msgstr ""
+
+#: rules.py:63
+msgid "Starting time"
+msgstr ""
+
+#: rules.py:64
+msgid "Ending time"
+msgstr ""
+
+#: rules.py:84
+msgid "Time Rule"
+msgstr ""
+
+#: rules.py:88
+msgid "These users visit between"
+msgstr ""
+
+#: rules.py:89
+msgid "{} and {}"
+msgstr ""
+
+#: rules.py:106
+msgid "Monday"
+msgstr ""
+
+#: rules.py:107
+msgid "Tuesday"
+msgstr ""
+
+#: rules.py:108
+msgid "Wednesday"
+msgstr ""
+
+#: rules.py:109
+msgid "Thursday"
+msgstr ""
+
+#: rules.py:110
+msgid "Friday"
+msgstr ""
+
+#: rules.py:111
+msgid "Saturday"
+msgstr ""
+
+#: rules.py:112
+msgid "Sunday"
+msgstr ""
+
+#: rules.py:136
+msgid "Day Rule"
+msgstr ""
+
+#: rules.py:152
+msgid "These users visit on"
+msgstr ""
+
+#: rules.py:153 rules.py:192 rules.py:400
+msgid "{}"
+msgstr ""
+
+#: rules.py:168
+msgid "Regex string to match the referer with"
```

Comment:
"Regex string to match the referer with" -> "Regular expression to match the referrer"